From 8519fe37c31ea59a29783b5749381db41293b504 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 17 Nov 2021 16:46:08 +0100
Subject: [PATCH 0211/1695] dts: stm32: add the tzc400 node for stm32mp15
 platforms

Add the TZC400 node reference into the STM32MP151 device
tree.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 755e814641492bea0e210eb48f2d22a207074ba8 dts: stm32: add the tzc400 node for stm32mp15 platforms]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: I6bc466bb3d128dbfb148e3ec7adab84d88204e2a
---
 core/arch/arm/dts/stm32mp151.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp151.dtsi b/core/arch/arm/dts/stm32mp151.dtsi
index dce40957d..9de6f1f09 100644
--- a/core/arch/arm/dts/stm32mp151.dtsi
+++ b/core/arch/arm/dts/stm32mp151.dtsi
@@ -1637,6 +1637,14 @@
 			};
 		};
 
+		tzc400: tzc@5c006000 {
+			compatible = "st,stm32mp1-tzc";
+			reg = <0x5c006000 0x1000>;
+			interrupts = <GIC_SPI 4 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&rcc TZC1>, <&rcc TZC2>;
+			st,mem-map = <0xc0000000 0x40000000>;
+		};
+
 		etzpc: etzpc@5c007000 {
 			compatible = "st,stm32-etzpc";
 			reg = <0x5C007000 0x400>;
-- 
2.47.2

